packages/echarts/src/index.ts

```diff
@@ -81,10 +81,9 @@ export const generateBasicAnnotations = (
       onboardingMessages = heatmapFactory(contextKey, chart, coords, visElement)
       break
 
-    default:
-      throw new Error(
-        `No onboarding for visualization type ${visType} available.`
-      )
```

Comment:
we should actually keep this default and additionally add e.g. "You can use "Generic" to initialize an empty visAhoi instance"

Reply:
Added default message also  'No onboarding for visualization type ${visType} available. Please use visualization type 'GENERIC' to create your own onboardings.'